sync/syncer.go

```diff
@@ -91,6 +91,11 @@ type Syncer struct {
 	clock             timesync.LayerTimer
 	exit              chan struct{}
 	currentLayerMutex sync.RWMutex
+
+	//todo fetch server
+	blockQueue *validationQueue
+	txQueue    *validationQueue
```

Comment:
is validation queue able to handle txs? I see it only handles blocks

Reply:
you are reviewing a work in progress and it is totally redundant doing so without talking to me first.
im just sketching out the solution youll find lots of things you don't understand...